include/boost/geometry/geometries/polyhedron.hpp

```diff
@@ -0,0 +1,97 @@
+#ifndef BOOST_GEOMETRY_GEOMETRIES_POLYHEDRON_HPP
+#define BOOST_GEOMETRY_GEOMETRIES_POLYHEDRON_HPP
+
+#include <memory>
+#include <vector>
+
+#include <boost/concept/assert.hpp>
+
+#include <boost/geometry/geometries/concepts/point_concept.hpp>
+#include <boost/geometry/geometries/ring.hpp>
+#include <boost/geometry/core/tag.hpp>
+#include <boost/geometry/core/tags.hpp>
+
+#include <boost/config.hpp>
+#ifndef BOOST_NO_CXX11_HDR_INITIALIZER_LIST
+#include <initializer_list>
+#endif
+
+namespace boost { namespace geometry 
+{
+
+namespace model
+{
+
+
+template 
+<
+    typename Point,
+    bool ClockWise = true, bool Closed = true,
```

Comment:
What is the purpose of `Clockwise`?
What is the purpose of `Closed`?

If you add closed clockwise rings as surfaces, isn't that enough?

What is the purpose of the pointlist?

Reply:
I found similar type of implementation for polygon, so I implemented that for Polyhedron. 
`Pointlist`, `Closed` and `Clockwise` are the parameter for the ring.

Reply:
`ClockWise` doesn't make any sense in 3D, right? You have a set of oriented rings so it makes sense to talk about consistent orientation or not. 

`Closed` makes sense since you may have a closed polyhedron (that bounds a solid with finite volume) or an open polyhedron with infinite volume.

Reply:
If those parameters are the ring parameters then why you do not use more descriptive names like `RingClockWise` ? Then, you assume that all rings will have the same order (e.g. clockwise), and all of them will be closed etc. which sounds problematic.